script/genpb2.sh

```diff
@@ -2,17 +2,17 @@
 #
 # Copyright 2019 Siemens AG
 #
-#   Licensed under the Apache License, Version 2.0 (the "License");
-#   you may not use this file except in compliance with the License.
-#   You may obtain a copy of the License at
+#   Licensed under the Apache License, Version 2.0 (the "License"); you may not
```

Comment:
Please, "domesticate" your IDE not to make style changes. Specially to licenses. It simplifies reviews a lot.

Reply:
that was a case of problem between chair and keyboard, I'll clean this up so the license doesn't get messed up.

Reply:
It wasn't messed up. But being a license, I had to check it more carefully than otherwise :slightly_smiling_face: